src/ml_board/backend/restful_api/data_access.py

```diff
@@ -236,6 +240,70 @@ def get_experiment_config(self, grid_search_id: str, experiment_id: str, config_
         else:
             raise InvalidPathError(f"File path {requested_full_path} is not safe.")
 
+    def get_checkpoint_dict_epoch(self, grid_search_id: str, experiment_id: str, epoch: str):
```

Comment:
return type information missing

Reply:
the documentation has been updated but I exactly am not sure what return-type information is missing. Could you help me out here?

Reply:
it would be `List[Dict]`, no?

Reply:
fixed and pushed